float32 input_voltage
float32 fan_speed
float32 temperature
PowerRegulatorChannelState[4] channels